app/docs/0.2.x/getting-started/introduction.md

```diff
@@ -8,12 +8,19 @@ title: Welcome to Kong
   <strong>Before you start:</strong> Make sure you've <a href="/install/">installed Kong</a> &mdash; It should only take a minute!
 </div>
 
-Now that you know what Kong is and how it works, this guide is going to take you through a quick introduction to Kong. We'll teach you how to use Kong to:
+Before going further into Kong, make sure you understand its [purpose and philosophy](/about). Once you are confident with the concept of API Gateways, this guide is going to take you through a quick introduction on how to use Kong and perform basic operations such as:
 
-- Running your own Kong instance.
-- Adding and consuming APIs
-- Installing plugins on Kong.
-- *And much more!*
+- [Running your own Kong instance][quickstart].
+- [Adding and consuming APIs][adding-your-api].
+- [Installing plugins on Kong][enabling-plugins].
+
+### What is Kong, technically?
+
+You probably read or heard that Kong was built on top of Nginx. But how is that possible exactly?
+
+Kong is a set of Lua scripts running in Nginx. Running Lua scripts in Nginx is made possible by the [lua-nginx-module](https://github.com/openresty/lua-nginx-module), which is bundled in [OpenResty](https://openresty.org/), along many others. Hence, Kong *is* an OpenResty application with powerful Lua scripts.
+
+Together, those scripts set the foundations for a pluggable architecture, where other scripts (so-called plugins) can be executed at runtime during the lifecycle of a request. Because of this pluggable nature, we like to think of Kong as **a paragon of microservice architecture**: at its core, it consists of no more than a couple thousands lines of code, handling database abstraction, routing and plugin management. Plugins can live in different code bases and inject themselves anywhere into the lifecycle of a proxied request in a few lines of code.
```

Comment:
```
The features and functionalities available in Kong are grouped into "Plugins" executed during the lifecycle of a request., providing a pluggable architecture and extendability with Lua.

Because of this pluggable nature, we like to think of Kong as **a paragon of microservice architecture**: at its core, it consists of no more than a couple thousands lines of code, handling database abstraction, routing and plugin management. Plugins can live in different code bases and inject themselves anywhere into the lifecycle of a proxied request in a few lines of code.
```